src/isomorphism/isoclasses.c

```diff
@@ -825,62 +2755,104 @@ int igraph_isoclass_create(igraph_t *graph, igraph_integer_t size,
     long int code;
     long int pos;
 
-    if (size < 3 || size > 4) {
-        IGRAPH_ERROR("Isoclasses are supported only three- and four-vertex graphs.",
-                     IGRAPH_UNIMPLEMENTED);
-    }
-
     IGRAPH_VECTOR_INIT_FINALLY(&edges, 0);
 
 #define CHECK_ISOCLASS(number, directed, size) \
     IGRAPH_ERRORF( \
-        "Isoclass " IGRAPH_PRId " requested, but there are only " \
-        IGRAPH_PRId " %s graphs of size " IGRAPH_PRId ".", IGRAPH_EINVAL, \
+        "Isoclass %" IGRAPH_PRId " requested, but there are only %" \
+        IGRAPH_PRId " %s graphs of size %" IGRAPH_PRId ".", IGRAPH_EINVAL, \
         (igraph_integer_t) number, directed ? "directed" : "undirected", (igraph_integer_t) size)
```

Comment:
The total number of graphs of a given size is missing from the argument list of `IGRAPH_ERRORF`.